### PR TITLE
added recipe for openslide-python

### PR DESCRIPTION
**Pull request by sarthakpati**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sarthakpati** [comment]
I have put in PR to remove openslide-python from bioconda and move it under conda-forge: https://github.com/bioconda/bioconda-recipes/pull/28978

**Comment by sarthakpati** [comment]
@conda-forge/help-python this is ready for merge!

**Comment by bgruening** [comment]
Thanks for the ping. I don't mind where it is living as long as it's maintained :)

File: recipes/openslide-python/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 00f5e34dca56500b598454107a0acf243788ea2072b5468b3fea59f756b832a8
+
+build:
+  number: 0
+  skip: true   # [py==30 or py==31 or py==32]
```
Comment (chrisburr) [inline recipes/openslide-python/meta.yaml:14]:
```suggestion
  skip: true   # [win]
```

File: recipes/openslide-python/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
```
Comment (chrisburr) [inline recipes/openslide-python/meta.yaml:15]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

**Comment by sarthakpati** [comment]
@bgruening do you mind if I add you as a maintainer to this?

**Pull request merged**